lnrpc/routerrpc/router.proto

```diff
@@ -7,41 +7,56 @@ package routerrpc;
 option go_package = "github.com/lightningnetwork/lnd/lnrpc/routerrpc";
 
 message PaymentRequest {
-    /**
-    A serialized BOLT-11 payment request that contains all information
-    required to dispatch the payment. If the pay req is invalid, or expired,
-    an error will be returned.
-    */
-    string pay_req = 1;
+    /// The identity pubkey of the payment recipient
+    bytes dest = 1;
 
+    /// Number of satoshis to send.
+    int64 amt = 2;
+
+    /// The hash to use within the payment's HTLC
+    bytes payment_hash = 3;
+ 
     /**
-    An absolute limit on the highest fee we should pay when looking for a route
-    to the destination. Routes with fees higher than this will be ignored, if
-    there are no routes with a fee below this amount, an error will be
-    returned.
+    The CLTV delta from the current height that should be used to set the
+    timelock for the final hop.
     */
-    int64 fee_limit_sat = 2;
+    int32 final_cltv_delta = 4;
 
     /**
-    An absolute limit on the cumulative CLTV value along the route for this
-    payment. Routes with total CLTV values higher than this will be ignored,
-    if there are no routes with a CLTV value below this amount, an error will
-    be returned.
+    A bare-bones invoice for a payment within the Lightning Network.  With the
+    details of the invoice, the sender has all the data necessary to send a
+    payment to the recipient. The amount in the payment request may be zero. In
+    that case it is required to set the amt field as well.
```

Comment:
can it be an empty string?

Reply:
Comment was copied from main rpc. Empty payment request explanation added.